package.json

```
@@ -79,7 +79,7 @@
     }
   },
   "package-deps": [
-    "linter:2.1.1"
+    "linter:3.0.0"
```

```suggestion
    "linter:2.1.1"
```
`linter@3` isn't required at all, `message.key` was added to `linter` _waaay_ back in v2 [here](https://github.com/steelbrain/linter/commit/aa4c48d84546bbb1abe47674364d1bb87f2ed3ae).

(Obviously don't just accept this change, simply remove the commit that added this.)

---

Fixed in https://github.com/AtomLinter/atom-minimap-linter/pull/82/commits/c267b22e5be996e0cbf3c979ae55bb51d4ad77fa